projects/angular/src/layout/nav/header.ts

```diff
@@ -102,10 +116,9 @@ export class ClrHeader implements OnDestroy {
     this.responsiveNavService.closeAllNavs();
   }
 
-  // toggles the nav that is open
-  toggleNav(navLevel: number) {
-    this.openNavLevel = this.openNavLevel === navLevel ? null : navLevel;
-    this.responsiveNavService.sendControlMessage(ResponsiveNavCodes.NAV_TOGGLE, navLevel);
+  openNav(navLevel: number) {
```

Comment:
Renaming this method is a breaking change. Why do we need it to say "open" vs. "toggle"?

If that is something that _must_ happen, we should make toggle an alias to `openNav()` and make a note that it is deprecated...

Reply:
Good catch! Will do

The thing is that we don't want to toggle it but just send an `open` event to the responsive nav service cause the hamburger will need to send only `open` event. The close button inside the nav-level will trigger the close.